tools/commands/build.py

```diff
@@ -31,9 +29,32 @@ def __str__(self):
     def clean(self):
         if os.path.exists(self.build_path):
             logger.info("cleaning {}".format(self.name))
-            os.system("xargs rm < {}/install_manifest.txt".format(self.build_path))
             shutil.rmtree(self.build_path)
 
+    def distclean(self):
```

Comment:
Probably wrong design on my part in the first place, distclean deletes the entire build directory, so that `maptools build  <all/framework/common/whatever> -c distclean` ignores the module name and simply deletes the whole build directory (and its subfolders - install amongst them). I suggest to either keep it as it was, so users accustomed to this command are not confused.